src/app/icons/icons.module.ts

```diff
@@ -177,6 +181,7 @@ const icons = {
   XSquare,
   Edit2,
   Twitter,
+  MyVote: CheckCircle,
```

Comment:
Reimporting `CheckCircle` with another name (`MyVote`), since the imported one is overwritten globally in `styles.scss`